Test Clang container

milljm opened the pull request:
Testing the Rocky-9 Clang Container

Refs #24793

Commit:
Test Clang container

Testing the Rocky-9 Clang Container

Refs #24793

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/1608799/) on eeb8764 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/24794/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/1608817/) on eeb8764 wanted to post the following:


## Framework coverage

Coverage did not change


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/contact)
- [`electromagnetics`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/electromagnetics)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/geochemistry)
- [`heat_conduction`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/heat_conduction)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/navier_stokes)
- [`optimization`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/optimization)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/richards)
- [`scalar_transport`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/scalar_transport)
- [`solid_properties`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/solid_properties)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/stochastic_tools)
- [`tensor_mechanics`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/tensor_mechanics)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/24794/coverage/xfem)

</details>



This comment will be updated on new commits.

Commit:
Don't mix compiler libs

We can't build everything up to this point using system libs (or
now a manual LLVM build), and then shimmy Conda's compiler libs
in front.

Move matplotlib from moose-test-tools to moose-tools

Configure Apptainer to install moose-test-tools when using Clang.

moosebuild commented:
All [jobs](https://civet.inl.gov/event/131799/) on 41fe00c : canceled by @milljm

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1617146/) on a86dc20 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1617866/) on c5626d7 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1617949/) on 317852f wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1618904/) on c2d7304 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1643288/) on 59ebbf4 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1644381/) on c96150a wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1644576/) on 0ee5a8f wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1646469/) on ecdef1e wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1646579/) on d6c2be7 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

milljm commented:
@loganharbour

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1646641/) on 90345c6 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1648531/) on 88707a9 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

Commit:
add matplotlib

rebase must have removed it as I was solving for conflicts.

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1771376/) on df296f4 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/tools

The following file(s) are unchanged:

  conda/mpich/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed
